services/fastfile_peeker/fastfile_peeker.rb

```diff
@@ -0,0 +1,55 @@
+require_relative "../../fastfile-parser/fastfile_parser"
+
+require "digest"
+
+module FastlaneCI
+  # Utility class designed for parsing Fastfiles in a repo.
+  class FastfilePeeker
+    class << self
+      def cache
+        @cache ||= {}
```

Comment:
This could just be 
```ruby
@_cache ||= {}
```

Reply:
Just for convention, I thought somewhere we discussed about making explicit `return` statements to improve code readability